cs/azure/Trainer/CountingCheckpointPolicy.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace VowpalWabbit.Azure.Trainer
```

Comment:
excellent point... missing more code now

---
In reply to: [66970805](https://github.com/JohnLangford/vowpal_wabbit/pull/1013#discussion_r66970805) [](ancestors = 66970805)